src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/integration/EligibilityAndSuitabilityCaseViewResourceIntTest.kt

```diff
@@ -79,7 +79,7 @@ class EligibilityAndSuitabilityCaseViewResourceIntTest : SqsIntegrationTestBase(
         .exchange()
         .expectStatus()
         .isOk
-        .expectBody().json(serializedContent("eligibility-and-suitability-view"), true)
```

Comment:
the overload that takes a boolean is deprecated

Reply:
does it default to strict checking?

Reply:
nope it doesn't and now I've just looked it's not deprecated in the version we're using. I must have done this when I was looking at upgrading dps spring boot plugin. I've reverted.